### PR TITLE
{devel}[GCCcore/8.3.0] Bazel v0.29.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d5a0959ed2380a9d990a7cc53aadb85c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2410.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1b888227e02d2132186f77413f5aca79 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b338e32e3b6e2cf1fd8d0a565503ed05 for a full test report.

File: easybuild/easyconfigs/b/Bazel/Bazel-0.29.1-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name = 'Bazel'
+version = '0.29.1'
+
+homepage = 'http://bazel.io/'
```
Comment (Micket) [inline easybuild/easyconfigs/b/Bazel/Bazel-0.29.1-GCCcore-8.2.0.eb]: https

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/cf284adb2e459fa9406c7aefbdb71c50 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ef5a8fff386610bb6b0b07f02b75b0af for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/994f4435286ccbc591072da06428730e for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4b3795c0c8bfd52832dc08f162628894 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Comment by Flamefire** [comment]
It seems Bazel now needs Python3: 

```
Error occurred while attempting to use the default Python toolchain (@rules_python//python:
autodetecting_toolchain).
According to '/usr/bin/python -V', version is 'Python 2.7.5', but we need version 3. PATH is:

/sw/installed/Java/11.0.2:/sw/installed/Java/11.0.2/bin:/sw/installed/binutils/2.31.1-GCCcore-8.2.0/bin:/sw/installed/GCCcore/8.2.0/bin:/sw/installed/EasyBuild/4.0.1/bin:/usr/lib64/qt-3.3/bin:/sw/taurus/tools
/slurmtools/default/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/opt/ibutils/bin:/home/s3248973/.local/bin

Please ensure an interpreter with version 3 is available on this platform as 'python3' or 'python', or else register an appropriate Python toolchain as per the documentation for py_runtime_pair (https://githu
b.com/bazelbuild/rules_python/blob/master/docs/python.md#py_runtime_pair).

Note that prior to Bazel 0.27, there was no check to ensure that the interpreter's version matched the version declared by the target (#4815). If your build worked prior to Bazel 0.27, and you're sure your ta
rgets do not require Python 3, you can opt out of this version check by using the non-strict autodetecting toolchain instead of the standard autodetecting toolchain. This can be done by passing the flag `--ex
tra_toolchains=@rules_python//python:autodetecting_toolchain_nonstrict` on the command line or adding it to your bazelrc.
----------------
Note: The failure of target //src:create_embedded_tools (with exit code 1) may have been caused by the fact that it is running under Python 3 instead of Python 2. Examine the error to determine if that appears to be the problem. Since this target is built in the host configuration, the only way to change its version is to set --host_force_python=PY2, which affects the entire build.

If this error started occurring in Bazel 0.27 and later, it may be because the Python toolchain now enforces that targets analyzed as PY2 and PY3 run under a Python 2 and Python 3 interpreter, respectively. See https://github.com/bazelbuild/bazel/issues/7899 for more information.
----------------
```

**Comment by boegel** [comment]
@Flamefire Let's follow up on that via #9222

**Pull request merged**